### PR TITLE
VS2015 extension installer

### LINKED ISSUE #2815: VS 2015 support
It seems VS 2015 doesn't support addins (.Addin) anymore and only uses vsix style extensions.

I didn't find an issue for it here yet, so here it is.

Probably have to package the VS addin as a vsix extension and change the installer.

### PR DESCRIPTION
**Pull request by KindDragon**
Fix #2815

### CODE REVIEW

**Comment by KindDragon** [comment]
Please test it https://drive.google.com/drive/folders/0B8UPsaxEuQgqWnA3X2tmMjMwNzA?usp=sharing

**Comment by jbialobr** [comment]
The settings button does not work.

**Comment by jbialobr** [comment]
VS2015 extension is not uninstalled along with GitExt being uninstalled.

**Comment by jbialobr** [comment]
>VS2015 extension is not uninstalled along with GitExt being uninstalled.

I take it back. I had installed a version of the extension compiled before your PR. After uninstalling it manually, the installer uninstalls the extension correctly.

**Comment by KindDragon** [comment]
> The settings button does not work.

Where is that button? On toolbar?

**Comment by jbialobr** [comment]
Yes.

File: Setup/Product.wxs [hunk 1]
```diff
@@ -198,6 +203,10 @@
                     <RegistryValue Value="&quot;[INSTALLDIR]GitExtensions.exe&quot; %1" Type="string" />
                 </RegistryKey>
             </Component>
+            <Component Id="VS2015_VSIX" Guid="*">
+                <VSExtension:VsixPackage File="GitExtensionsVSIX.vsix" PackageId="GitExtensions..3166f76a-2b07-410a-a72a-509d6a2d146c" Target="community" TargetVersion="14.0" Vital="yes" Permanent="no" VsixInstallerPathProperty="VS2015_VSIX_INSTALLER_PATH" />
+                <File Source="..\GitExtensionsVSIX\bin\Release\GitExtensionsVSIX.vsix" />
```
Comment (ashumkin) [inline Setup/Product.wxs]: Arkady @KindDragon, how can I get `GitExtensionsVSIX\bin\Release\GitExtensionsVSIX.vsix` file?
I'm trying to make an installer, but finally `MakeInstallers.cmd` fails missing this file. I have no enough knowledge to guess how to get the file appear as I'm not a VS user. Thank you!

Comment (KindDragon) [inline Setup/Product.wxs]: Maybe you install VS without "Visual Studio Extensibility Tools"? https://msdn.microsoft.com/en-us/library/bb166441(v=vs.140).aspx#Anchor_0

Comment (ashumkin) [inline Setup/Product.wxs]: Definitely!
Moreover, I haven't even installed VS. I use MSBuild (and VS build tools) to build the project (and the installer, too).
So, the question is "how to generate/build `.vsix` file?" How are they (`.vsix`-files) generated?

Comment (KindDragon) [inline Setup/Product.wxs]: I don't think that this is possible without "Visual Studio Extensibility Tools"

Comment (KindDragon) [inline Setup/Product.wxs]: You can use VS2015 Community Edition

**Pull request merged**